lnst/RecipeCommon/Perf/Measurements/Results/LinuxPerfMeasurementResults.py

```diff
@@ -1,6 +1,11 @@
+from typing import TYPE_CHECKING
+
 from lnst.Controller.Host import Host
 from lnst.RecipeCommon.Perf.Measurements.Results.BaseMeasurementResults import BaseMeasurementResults
 
+if TYPE_CHECKING:
+    from lnst.RecipeCommon.Perf.Measurements.LinuxPerfMeasurement import LinuxPerfMeasurement
+
```

Comment:
Are you sure this is necessary? We do not do type checking in CI. Why do we need this?

Reply:
Alright, I'm removing this.